gulpfile.js

```diff
@@ -4,109 +4,53 @@
  * importing Shoreditch Partials and minifying the css file to .min.css
  *
  * Tasks
- * default : Runs SASS task
+ * default : Runs sass and test task
+ * sass:sync : Compiles civicase.scss under scss folder to CSS counterpart
```

Comment:
Instead of having the doc on the top, they should be moved on top of the actual tasks.
```
/**
 * Updates and sync the scssRoot paths
 */
gulp.task('sass:sync', sassSyncTask);
```
Otherwise we are duplicating the description.
Please check all cases.